### PR TITLE
Implement UI for online count

### LINKED ISSUE #9: Design Interface + Connect Number of Online Counts
Issue 1C

- Estimated Effort: 5 days
- Dependencies: none
- Initial Assignments: @vanessalin0226 
- Milestones: Completed in Sprint 1
- Acceptance Criteria:
  - Displays correct online counts on UI at right places

### PR DESCRIPTION
**Pull request by vanessalin0226**
Addresses Issue #9 

- Added UI design and placeholders for online user count for the User and Group page
- Changes made in following files:
  - `themes/nodebb-theme-persona/templates/partials/users_list_menu.tpl` (User page)
  - `themes/nodebb-theme-persona/templates/partials/groups/userOnlineCount.tpl` (Group page)
  - `themes/nodebb-theme-persona/templates/partials/groups/memberlist.tpl` (Within Group page, member list area)
- Files added:
  - `themes/nodebb-theme-persona/templates/partials/groups/userOnlineCount.tpl` (Count for User page)
  - `themes/nodebb-theme-persona/templates/partials/groups/groupOnlineCount.tpl` (Count for Group and within Group page)
- See attached photos for details

Screenshot 2023-09-22 at 2 48 57 PM
Screenshot 2023-09-22 at 2 49 02 PM
Screenshot 2023-09-22 at 2 49 06 PM

### CODE REVIEW

File: themes/nodebb-theme-persona/templates/partials/groups/userOnlineCount.tpl [hunk 1]
```diff
@@ -0,0 +1 @@
+<h6> ({groups.onlineUserCount}10) </h6>
```
Comment (cwesterink) [inline themes/nodebb-theme-persona/templates/partials/groups/userOnlineCount.tpl]: The 10 should be removed. 
Not a big deal. Can be removed in another PR if necessary.

**Review by cwesterink — APPROVED** [review]
LGTM

File: themes/nodebb-theme-persona/templates/partials/groups/userOnlineCount.tpl (hunk 1 above)
Comment (vanessalin0226) [reply themes/nodebb-theme-persona/templates/partials/groups/userOnlineCount.tpl]: I just removed the 10 now, it's just a placeholder so something shows up on the UI.
